### DISCUSSION TITLE
PR comments don't get updated

### DISCUSSION THREAD

**Discussion by codeforcesmeme** [vercel/vercel#5163] [Help]
We've just moved to vercel at our company but there is a problem that PR comments don't get updated **ever**
I have two scenarios:
- I push a new branch and wait for the build to finish then create a PR, vercel bot comments 
```
This pull request is being automatically deployed with Vercel (learn more).
To see the status of your deployment, click below or on the icon next to each commit.

🔍 Inspect: `{url}`
✅  Preview: `{url}`
```
But on any additional commit the second url doesn't change to `In progress` then the new url and latest commit deployed, like I've seen in other repos.

- I push a new branch then create a PR immediately the bot comments 
```
This pull request is being automatically deployed with Vercel (learn more).
To see the status of your deployment, click below or on the icon next to each commit.

🔍 Inspect: `{url}`
✅  Preview: In Progress
``` 
And the comment doesn't get updated ever, even though the build is done and we see in the PR timeline `@vercel bot deployed to Preview 17 hours ago`.

**Comment by paulogdm** [comment]
Hello,

Please use https://vercel.com/support/request so we can look into it.
